.travis/snap.sh

```diff
@@ -9,11 +9,15 @@ git config user.name "CI Bot"
 git config user.email "rocketchat.buildmaster@git.launchpad.net"
 
 # Determine the channel to push snap to.
-if [[ $TRAVIS_TAG ]]
- then
+if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+ ]]; then
+    CHANNEL=candidate
+    RC_VERSION=$TRAVIS_TAG
+elif [[ $TRAVIS_TAG ]]; then
     CHANNEL=stable
+    RC_VERSION=$TRAVIS_TAG
 else
     CHANNEL=edge
+    RC_VERSION=0.55.0-develop
```

Comment:
$TRAVIS_TAG does not exists on the develop branch.. :( 

Lets merge this as it is and think about a solution after that